UnitTests/Static_HelperTests.cs

```diff
@@ -0,0 +1,181 @@
+using ClassLibrary1;
+using NUnit.Framework;
+using System;
+
+namespace UnitTests
+{
+    public class Static_HelperTests
```

Comment:
In general fix formatting:
1. empty lines between AAA sections
2. empty lines after `{}`